<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LFG Magic</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/file.ico</normaloff>:/file.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>20</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>North America</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Europe</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>20</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>Story</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Explorable</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>20</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>Ascalonian Catacombs</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Caudecus's Manor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Twilight Arbor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sorrow's Embrace</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Citadel of Flame</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Honor of the Waves</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Crucible of Eternity</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>The Ruined City of Arah</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fractals of the Mists</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>41</y>
      <width>661</width>
      <height>381</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>661</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="gridStyle">
     <enum>Qt::DotLine</enum>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Comment</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Time</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#0000ff;&quot;&gt;Sound off&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>430</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#0000ff;&quot;&gt;Create new LFG&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRefresh">
    <property name="title">
     <string>Refresh</string>
    </property>
    <addaction name="action10_sec"/>
    <addaction name="action15_sec"/>
    <addaction name="action30_sec"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <addaction name="actionSeconds"/>
    <addaction name="actionMinutes"/>
   </widget>
   <widget class="QMenu" name="menuCopy_style">
    <property name="title">
     <string>Copy style</string>
    </property>
    <addaction name="action_player_name"/>
    <addaction name="action_join_player_name"/>
   </widget>
   <widget class="QMenu" name="menuDonate">
    <property name="title">
     <string>Donate…</string>
    </property>
    <addaction name="action_to_creator_of_gw2lfg_com"/>
    <addaction name="action_to_creator_of_LFG_Magic"/>
    <addaction name="separator"/>
    <addaction name="actionWhat_s_the_difference"/>
   </widget>
   <addaction name="menuRefresh"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuCopy_style"/>
   <addaction name="menuDonate"/>
  </widget>
  <action name="action10_sec">
   <property name="text">
    <string>10 sec.</string>
   </property>
  </action>
  <action name="action15_sec">
   <property name="text">
    <string>15 sec.</string>
   </property>
  </action>
  <action name="action30_sec">
   <property name="text">
    <string>30 sec.</string>
   </property>
  </action>
  <action name="actionSeconds">
   <property name="text">
    <string>Seconds</string>
   </property>
  </action>
  <action name="actionMinutes">
   <property name="text">
    <string>Minutes</string>
   </property>
  </action>
  <action name="action_player_name">
   <property name="text">
    <string>&lt;%name%&gt;</string>
   </property>
  </action>
  <action name="action_join_player_name">
   <property name="text">
    <string>&lt;/join %name%&gt;</string>
   </property>
  </action>
  <action name="action_to_creator_of_gw2lfg_com">
   <property name="text">
    <string>…to creator of gw2lfg.com</string>
   </property>
  </action>
  <action name="action_to_creator_of_LFG_Magic">
   <property name="text">
    <string>…to creator of LFG Magic</string>
   </property>
  </action>
  <action name="actionWhat_s_the_difference">
   <property name="text">
    <string>What's the difference?</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
